app/views/staff/websites/_form.html.haml

```diff
@@ -9,7 +9,7 @@
       - if website.background.attached?
         = f.label :background, "Current Background", class: "control-label"
         %div
-          = image_tag(website.background)
+          = image_tag(website.background, width: "200px")
```

Comment:
On my latest branch I did a similar thing using resize. By the way, it is good to know about these resize helpers if you don't already know about them: https://bloggie.io/@kinopyo/upgrade-guide-active-storage-in-rails-6

Reply:
Per ping, will leave this as a width resize for now due to resize throwing errors on SVG. We can take another stab at this if we don't like how it is working.